### PR TITLE
Improved: Add Diagonal Brakes to LSM Launched Roller Coaster

### PR DESCRIPTION
**Pull request by karst**

### CODE REVIEW

**Comment by karst** [comment]
This is also ready now.

File: distribution/changelog.txt [hunk 1]
```diff
@@ -35,7 +35,7 @@
 - Fix: [#23015] Crash when loading a save game when the construction window is still open.
 - Fix: [#23018] Crash when loading a new game when the construction window is still open.
 - Fix: [#23023] Large scenery clearance height interpreted as negative when greater than 127.
-- Fix: [#23044] “remove_unused_objects” command causes blank peep names.
+- Fix: [#23044] "remove_unused_objects" command causes blank peep names.
```
Comment (Gymnasiast) [inline distribution/changelog.txt:38]: Accidental change?

**Comment by Gymnasiast** [comment]
I’m okay with including these in v0.4.17. It is a mostly cosmetic change (just adding drawing code), so I think the dog-eating potential of it is rather limited.

**Review by Gymnasiast — APPROVED** [review]
Tested it and seems to work correctly.

**Pull request merged**